app/src/main/java/fr/free/nrw/commons/media/WikidataMediaClient.kt

```diff
@@ -0,0 +1,93 @@
+package fr.free.nrw.commons.media
+
+import fr.free.nrw.commons.BetaConstants
+import fr.free.nrw.commons.Media
+import fr.free.nrw.commons.category.ContinuationClient
+import fr.free.nrw.commons.explore.media.MediaConverter
+import io.reactivex.Single
+import org.wikipedia.dataclient.mwapi.MwQueryPage
+import org.wikipedia.dataclient.mwapi.MwQueryResponse
+import org.wikipedia.wikidata.Entities
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**
+ * Media Client to handle custom calls to Commons MediaWiki APIs of production server
+ */
+@Singleton
+class WikidataMediaClient @Inject constructor(
+    private val wikidataMediaInterface: WikidataMediaInterface,
+    private val mediaDetailInterface: MediaDetailInterface,
+    private val mediaConverter: MediaConverter
+) : ContinuationClient<MwQueryResponse, Media>() {
+
+    /**
+     * Fetch images for depict ID
+     *
+     * @param query depictionEntityId ex. Q9394
```

Comment:
Sorry for nitpicking, but would you mind writing it in the String format with quotes, for instance `"Q9394"`? Thanks